### PR TITLE
CDM-34: Create Navbar component

### PR DESCRIPTION
**Pull request by danielaprado**
### Description

- Create Navbar component
- Desktop & Mobile responsive

### Attachments
- **Desktop**
image
- **Mobile**
image
image

### Reviewers

- [x] @catarinapm 
- [ ] @mariasbl

### CODE REVIEW

File: fe/app/components/Navbar/MenuOptions/MenuOptions.tsx
Comment (catarinapm) [inline fe/app/components/Navbar/MenuOptions/MenuOptions.tsx:1]: Why don't you create a function with that repeated <Link> and pass the variables such as category and label?
Also, the onClick is suppose to be the same for every <Link>?

**Review by catarinapm — COMMENTED** [review]
I would suggest that in the mobile version the on press would make the backgound blue and the text would be white. Instead of underline as the Web version. We can discuss this!

**Review by catarinapm — CHANGES_REQUESTED** [review]
Also the navbar should be floating, right?

File: fe/app/components/Navbar/MenuOptions/MenuOptions.tsx
Comment (danielaprado) [reply fe/app/components/Navbar/MenuOptions/MenuOptions.tsx:1]: U right, doing it!
Yes they all go to to different routes (href) but when clicking on the option we want to close the menu

**Review by catarinapm — APPROVED** [review]
Thank youu!

**Review by mariasbl — APPROVED** [review]

**Pull request merged**